lib/soapy_cake/response.rb

```diff
@@ -22,6 +22,14 @@ def to_enum
       end
     end
 
+    def to_xml
```

Comment:
Okay yes, that works. I still think this is weird. Especially since `to_enum.count` parses the whole XML just to count the elements.